client/scripts/app.js

```diff
@@ -57,6 +66,13 @@ class Nametag extends Component {
       window.location = postAuth
       window.localStorage.removeItem('postAuth')
     }
+
+    // Attach user id to Sentry error reports
+    client.watchQuery({query: USER_QUERY}).subscribe({
```

Comment:
Sure -- I don't have a full view of the codebase so I anticipated this might come up in review. I looked around the code store but didn't spot a good place to put it there.

Does [`watchQuery`](http://dev.apollodata.com/core/apollo-client-api.html#ApolloClient.watchQuery) kick off a fetch to a query? The Apollo docs were a bit hard to understand, but I thought that it was akin to setting a watch on the data being fetched when it was triggered via other means.

If you'd like, feel free to put a commit on this branch moving this to where you'd like, or if you can point me in the right direction I'll do it.